i18n-populator.config.json

```diff
@@ -1,5 +1,6 @@
 {
   "basePath": "example",
+  "sort": true,
```

Comment:
I like this approach as it enforces all members of a team to have the same configuration.

Instead of using true and false we could use special words, in case in a future would be added more sort options, for example:

`"A-Z"` -> Alfabeticamente
`"Z-A"` -> Alfabeticamente invertido
`"none"` -> desactivado